modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Fix double bottom border on inline mode editor in `tinymce-5` skin. #TINY-9108
```

Comment:
Maybe not sure. Since the group is Fixed and I think other changelog items are worded like this instead.

```suggestion
- Double bottom border on inline mode editor in `tinymce-5` skin. #TINY-9108
```